### ISSUE TITLE
Not_Found error when using variable in init expression if autoremove

### ISSUE THREAD

**Issue by lbinria** [imitator-model-checker/imitator#51]
## Description

A `Not_Found` error is raised when we using a variable in a init expression (discrete or continuous section) that was previously removed by auto remove. This bug already exist in version 3.0.0.

Example : 
```
int i;
(* i is not used *)

b := i > 0
```

## Solution

Variables that are dependent of a used variable are keep.
